client/src/store/notifications/actions.ts

```diff
@@ -0,0 +1,39 @@
+import { Dispatch } from 'redux';
+import { showAlert } from '../alert/actions';
+import { SocketAPI } from '../../helpers/socket.connection';
+
+const socketNotifications = new SocketAPI('notifications');
+
+export const LIKE_NOTIFICATION = 'liked your post';
+export const COMMENT_NOTIFICATION = 'commented your post';
+export const FOLLOW_NOTIFICATION = 'started following your post';
```

Comment:
could you move these consts to notification config file with const strings from this file?

Reply:
I consider no need to do this, cause this is not belong to emitter
This is plain text for notification

Reply:
it is a good practice to use const instead of hardcode string in js or html. 
Why? If you need to change the text, you no need to search for file. You could just open config and change some text.

Reply:
realised